client/src/components/App.jsx

```diff
@@ -9,18 +10,22 @@ class App extends React.Component {
     this.state = {
       locationRatings: {},
       reviews: []
-    }
+    };
   }
 
-
   componentDidMount() {
     $.get(`http://localhost:1010/reviews/initial/${this.props.locationID}`)
       .done((results) => {
         this.setState({
           locationRatings: results.location[0],
           reviews: results.reviews
-        })
-      })
+        });
+      });
+  }
+
+  readMore (index) {
+    this.state.reviews[index].readMore = true;
+    this.setState({reviews: this.state.reviews});
```

Comment:
This will likely NOT trigger a rerender as intended. Becuase the reference to this.state.reviews is the same, React will not see this as a change.
You must replace reviews with a new object, from Object.assign for example.
Here are some options: https://www.freecodecamp.org/news/handling-state-in-react-four-immutable-approaches-to-consider-d1f5c00249d5/